Microsoft.Toolkit.Mvvm/ComponentModel/ViewModelBase.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#pragma warning disable SA1512
+
+// Original file header:
```

Comment:
I'm following up with our legal contacts for how we handle these scenarios in general; we may need to also copy this in the third party notice at the root of the project. I'll get clarity and report back.

Reply:
Perfect! Also not sure if this is relevant (I'm not a legal expert myself so I wouldn't know), but I didn't actually copy any of the original files from `MvvmLight`, every single file in this PR has been rewritten from scratch. It's just the structure in \*some\* classes that's similar (eg. `ViewModelBase`, though that too has things that are not in `MvvmLight`), while others (eg. the `Messenger` and various commands) are completely different. I thought I'd clarify this in case it could help! 😄

Reply:
@Sergio0694 talked to our legal experts. Since it's all original code written by yourself, we shouldn't have the official header here. However, since it's closely inspired. Please add this license to the ThirdPartyNotice in the root of the repo and just call that out as a note for an effective credit line.

Reply:
Awesome, thank you for following up on this! 😊

Made these changes in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3229/commits/0b36b2184eb9b1f4e7540f03533406071fa90984 and https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3229/commits/84eb8562827e40f9c8e792ab67e40633a0332ff6.